src/data-fetcher-loop/data-fetcher-loop.ts

```diff
@@ -73,10 +74,17 @@ export const runDataFetcher = async () => {
         .flat(2)
     );
 
+    // Compute the set of URLs coming from the config. These are trusted and don't need to be verified.
```

Comment:
The question is what to do if there is a same URL in both contract and config. It makes sense to treat it as trusted, so that is what the code does as well.

Reply:
I agree with your decision.